src/imitation/scripts/common/demonstrations.py

```diff
@@ -27,36 +27,17 @@ def fast():
     n_expert_demos = 1  # noqa: F841
 
 
-def guess_expert_dir(data_dir: str, env_name: str) -> str:
-    rollout_hint = env_name.rsplit("-", 1)[0].replace("/", "_").lower()
-    return os.path.join(data_dir, "expert_models", f"{rollout_hint}_0")
-
-
-@demonstrations_ingredient.config_hook
-def hook(config, command_name, logger):
-    """If rollout_path not set explicitly, then guess it based on environment name."""
-    del command_name, logger
-    updates = {}
-    if config["demonstrations"]["rollout_path"] is None:
-        data_dir = config["demonstrations"]["data_dir"]
-        env_name = config["common"]["env_name"].replace("/", "_")
-        updates["rollout_path"] = os.path.join(
-            guess_expert_dir(data_dir, env_name),
-            "rollouts",
-            "final.pkl",
-        )
-    return updates
-
-
 @demonstrations_ingredient.capture
-def load_expert_trajs(
```

Comment:
I'd like to maintain the option of loading rollouts from disk. Otherwise this feels like a regression in functionality, e.g. we'd have no way on training from demonstrations provided by a human being rather than a synthetic policy.